src/test/regress/expected/local_table_join.out

```diff
@@ -1466,7 +1466,48 @@ DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT count(*) AS c
    100
 (1 row)
 
+-- issue 4682
+create table tbl1 (a int, b int, c int, d int);
+INSERT INTO tbl1 SELECT i,i,i,i FROM generate_series(1,10) i;
+create table custom_pg_dist_rebalance_strategy(name text);
+INSERT INTO custom_pg_dist_rebalance_strategy values('a');
+-- try with local tables to make sure the results are same when tbl1 is distributed
+select COUNT(*) from
+  pg_operator
+  inner join tbl1 on (select 1 from custom_pg_type) >= d
+  left join custom_pg_dist_rebalance_strategy on 'a' = name
+where a + b + c > 0;
+ count
+---------------------------------------------------------------------
+   784
+(1 row)
+
+select create_distributed_table('tbl1', 'a');
+NOTICE:  Copying data from local table...
+DEBUG:  Copied 10 rows
+NOTICE:  copying the data has completed
+DETAIL:  The local data in the table is no longer visible, but is still on disk.
+HINT:  To remove the local data, run: SELECT truncate_local_data_after_distributing_table($$local_table_join.tbl1$$)
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- there is a different output in pg11 and in this query the debug messages are not
+-- as important as the others so we use notice
+set client_min_messages to NOTICE;
+select COUNT(*) from
+  pg_operator
+  inner join tbl1 on (select 1 from custom_pg_type) >= d
+  left join custom_pg_dist_rebalance_strategy on 'a' = name
+where a + b + c > 0;
+ count
+---------------------------------------------------------------------
```

Comment:
hmm, would this result rely on the size of `pg_operator `? So, maybe create a placeholder table instead to make it consistent output?

Reply:
Ahh, I haven't seen pg_operator :) I was wondering why it wasn't consistent with the other tables. Thanks!